src/client/actions/index.js

```diff
@@ -1,20 +1,21 @@
 // Action Types
-export const ADD_DOCTOR = 'ADD_DOCTOR';
+export const ADD_USER = 'ADD_USER';
 export const ERROR = 'ERROR';
 
 const url = 'http://localhost:3000/api';
 
 export const error = err => ({ type: 'ERROR', data: err });
 
-export const addDoctor = doctor => {
-  console.log('info for add doctor', doctor); 
-  return { type: ADD_DOCTOR, data: doctor };
+export const addUser = user => {
+  console.log('info for add user', user); 
+  return { type: ADD_USER, data: user };
 };
 
-export const postDoc = (name, dob, office, phone, sex) => {
-  const payload = JSON.stringify({ name, dob, office, phone, sex });
+export const postUser = (name, username, password, email, weight, bmi, goal, points) => {
```

Comment:
this function should be merged with addUser (i.e. remove postUser and move the content into addUser